src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -307,25 +307,23 @@ MPL_STATIC_INLINE_PREFIX void MPIDI_OFI_win_request_complete(MPIDI_OFI_win_reque
     }
 }
 
-MPL_STATIC_INLINE_PREFIX fi_addr_t MPIDI_OFI_comm_to_phys(MPIR_Comm * comm, int rank, int ep_family)
```

Comment:
I think that's a good idea, but that'll come later with multi-VNI support. I think @yfguo or @halimamer even prototyped it in a very similar way.
Purpose of this commit/PR was to achieve the same functionality as of right now.